python/cugraph/cugraph/structure/number_map.py

```diff
@@ -648,6 +666,31 @@ def unrenumber(self, df, column_name, preserve_order=False, get_column_names=Fal
                 mapping[nm] = nm + "_" + column_name
             col_names = list(mapping.values())
 
+        if isinstance(self.input_src_col_names, list):
+            input_src_col_names = self.input_src_col_names.copy()
+            input_dst_col_names = self.input_dst_col_names.copy()
+        else:
+            # Assuming the src and dst columns are of the same length
+            # if they are lists.
+            input_src_col_names = [input_src_col_names]
+            input_dst_col_names = [input_dst_col_names]
+        if not isinstance(col_names, list):
+            col_names = [col_names]
+
+        # FIXME: instead of hardcoded value, it should be 'simpleGraphImpl.srcCol'
+        # but there is no way to retrieve it with the current API
```

Comment:
I think the NumberMap instance is constructed with the src and dst column names (`self.src_col_names`, `self.dst_col_names`), can you use those?

Reply:
Right I didn't catch that those were also captured in NumberMap. Thanks

Reply:
But I think I can capture that in `NumberMap` from `simpleGraph`